community-plugins.json

```diff
@@ -692,10 +692,18 @@
         "repo": "d-sauer/Obsidian-Min3ditorHotkeys-plugin"
     },
     {
-       "id":"obsidian-jsonifier",
-       "name":"JSONifier",
-       "author":"Kjell Connelly",
-       "description":"JSON.stringify() or JSON.parse() highlighted text and copy to clipboard.",
-       "repo":"KjellConnelly/obsidian-jsonifier"
+        "id": "obsidian-jsonifier",

```

Comment:
This entry was indented with 3 spaces, just added an extra space before the keys. And another space between the keys and values.